### DISCUSSION TITLE
[iOS] Can’t close the Edit Status screen after using the emoji picker

### DISCUSSION THREAD

**Discussion by yakovmanshin** [community/community#124503] [Mobile] [answered]
### Select Topic Area

Bug

### Body

* Opening the emoji picker on the status-editing screen breaks the Cancel button used to close this screen
* See the [screen recording](https://github.com/community/community/assets/4648742/fb63b0cc-ffdf-4bb6-8041-c3d05a4d7685)
* You don’t actually have to pick an emoji—just opening and closing the picker is enough to break the button
* The screen can still be closed with a swipe
* GitHub Mobile (iOS) v1.163.0 (131249690)

**Comment by alcere** [comment, marked as answer]
Hi @yakovmanshin. Thank you for reporting this! The team was able to reproduce the issue and will do its best to ship a fix as soon as possible. Stay tuned for updates!
